code/modules/mob/living/carbon/human/human_defines.dm

```diff
@@ -30,11 +30,7 @@ var/global/default_martial_art = new/datum/martial_art
 	//Equipment slots
 	var/obj/item/wear_suit = null
 	var/obj/item/w_uniform = null
-	var/obj/item/shoes = null
 	var/obj/item/belt = null
-	var/obj/item/gloves = null
-	var/obj/item/clothing/glasses/glasses = null
-	var/obj/item/ears = null

```

Comment:
Moved all these to carbon level, to make some code a bit easier, despite no other carbon using those vars.